batch/batch/cloud/gcp/driver/create_instance.py

```diff
@@ -128,7 +129,7 @@ def scheduling() -> dict:
                 'boot': True,
                 'autoDelete': True,
                 'initializeParams': {
-                    'sourceImage': f'projects/{project}/global/images/batch-worker-13',
+                    'sourceImage': f'projects/{project}/global/images/batch-worker-parsa-100',
```

Comment:
@daniel-goldstein or I will need to actually build `batch-worker-14` and you'll need to change this to `batch-worker-14` when we're done.

Also, reminder for Daniel and I that we need to let the Australians create their own boot disk image before merging this PR.